### ISSUE TITLE
Cluster Domain name is not always cluster.local

### ISSUE THREAD

**Issue by vr** [grafana/k6-operator#233]
### Brief summary

I encounter an issue with k6-operator which is not able to fetch status of his runners, on a cluster which do not use `cluster.local` (this is uncommon config but perfectly valid) because of this hardcoded code https://github.com/grafana/k6-operator/blob/9a06624d1bb67fd2afc071ed99fc9419d630f369/controllers/k6_start.go#L20

What could be the way to fix this ? define some variable somewhere ? using `.svc` instead of `.svc.cluster.local` ? what do you prefer ?

as a workaround, i changed on which the cluster i deploy but could be great to fix this.

### k6-operator version or image

latest

### K6 YAML
```
apiVersion: k6.io/v1alpha1
kind: K6
metadata:
  name: k6-foo
spec:
  parallelism: 10
  arguments: --out statsd
  script:
    configMap:
      name: crocodile-stress-test
      file: test.js
  runner:
    env:
      - name: K6_STATSD_ENABLE_TAGS
        value: "true"
```

### Steps to reproduce the problem

deploy some K6 resources

### Expected behaviour

runners can be joined and status fetched

### Actual behaviour

runners can't be joined and nothing finish

**Comment by yorugac** [comment]
Hi @vr, thanks for opening the issue!

I think ideally, it'd be preferable to switch to using `service.namespace` format instead of the full domain as now, then the whole question about domains will disappear. There were several different cases raised around networking in the operator which sometimes seemed contradictory to each other, so there's an umbrella issue meant to hopefully resolve all of them at once: #186

**Comment by yorugac** [comment]
This should have been resolved in #288; please feel free to (re)open an issue if something else comes up!
